extensions/gluonTS/src/main/java/ai/djl/gluonTS/examples/DeepARExample.java

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
Move examples into top level example folder

Reply:
Got it!